src/Sulu/Component/DocumentManager/Event/RemoveLocaleEvent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Event;
+
+use PHPCR\NodeInterface;
+
+class RemoveLocaleEvent extends AbstractMappingEvent
+{
+    /**
+     * @param object $document
+     * @param string $locale
+     */
+    public function __construct($document, $locale)
+    {
+        $this->document = $document;
+        $this->locale = $locale;
+    }
+
+    /**
+     * Sets the node this event should operate on.
+     *
+     * TODO Check if should be move to DocumentManager itself
```

Comment:
for other events we also set the node from a subscriber - so feel free to remove this comment :)

Reply:
Yeah but on these events there is the same comment, should I remove all of them?